node.gyp

```diff
@@ -1083,6 +1083,7 @@
         'test/cctest/test_environment.cc',
         'test/cctest/test_linked_binding.cc',
         'test/cctest/test_platform.cc',
+        'test/cctest/test_per_process.cc',
```

Comment:
```suggestion
        'test/cctest/test_per_process.cc',
        'test/cctest/test_platform.cc',
```

Reply:
Oh yeah, it started off as `test_process.cc`...